src/ReflectionMethod.php

```diff
@@ -80,7 +80,8 @@ public function ___debugInfo()
      */
     public function __toString()
     {
-        $hasReturnType    = $this->hasReturnType();
+        $returnType       = $this->getReturnType();
+        $hasReturnType    = ($returnType !== null); // Internally same as: $this->hasReturnType();
```

Comment:
It came up in our discussion, and it wasn't obvious to me, so I added the comment.